gui/default/syncthing/core/syncthingController.js

```diff
@@ -802,7 +802,7 @@ angular.module('syncthing.core')
             if (status == 'paused') {
                 return 'default';
             }
-            if (status === 'syncing' || status === 'scanning') {
+            if (status === 'syncing' || status === 'preparing-sync' || status === 'scanning') {
```

Comment:
Why not just preparing?

Reply:
I considered that too generic, there might as well be a "preparing-scan" at some point.

Reply:
Perhaps them preparing-to-sync? Preparing-sync sounds like sync is a thing, ala preparing-dinner

Reply:
I'm for "Preparing" or "Preparing to sync" (small "s" like "Waiting to scan")

Reply:
I went for `sync-preparing` as I discovered a precedent for that in `scan-waiting` (and I really don't think the variable name matters that much).

And on the UI side I'd like to keep the connection to syncing - it's not just preparing anything, it's preparing a sync run. And regarding capitalization: Currently "Waiting to scan" is the outlier, everything else is in title case. Feel free to start a forum discussion if you'd like to change that (though probably not, as it requires retranslation for a minor detail).